frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -87,7 +88,10 @@ export const StrategyItemContainer: FC<StrategyItemContainerProps> = ({
             : ({ children }) => <> {children} </>;
 
     return (
-        <Box sx={{ position: 'relative' }}>
+        <Box
+            className={strategy.disabled ? disabledStrategyClassName : ''}
```

Comment:
This lets us easily do all kinds of style tweaks to disabled strategies without having to pass around "disabled" props everywhere.